src/Components/Redux/Mission/missionAPI.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+
+const url = 'https://api.spacexdata.com/v3/missions';
```

Comment:
Rockets and Missions data is being fetched successfully from SpaceX API 🤓

Reply:
@mateo951 Thank you for the approval